pages.de/common/clang.md

```diff
@@ -0,0 +1,24 @@
+# clang
```

Comment:
Optional fix (in the context of this PR, but mandatory in the context in another one): the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/clang.md) page have broken placeholders: [wrong path placeholder format is used](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md): should be fixed by somebody.